README.md

```diff
@@ -205,21 +246,45 @@ Each analysis definition is a map that has some properties common to all analysi
 >Note: only External program and Compound are available to all users. Other types require the Project Automation Framework developer license which is used for internal Qualisys development and by development partners.
 >
 - **Prerequisites**: A sequence of measurement type names and analysis names that has to be completed before this analysis can be run. Measurement types are considered complete for the current session when the user has made at least the number of measurements given by that measurement type’s Minimum count field. Analyses are considered complete when the file denoted by the Output file field exists.
+> Note: Prerequisites can't be used when executing parent session level analyses.
 - **Output file**: The name of a file that is created when this analysis is run. QTM uses this to check if the analysis has been completed. It is also used to issue a warning if this file has been changed since the analysis was last run for the current session. This property can contain patterns.
 
+- **Exclude:** Optional. A string or list of strings to specify the file names to exclude. The string patterns available are the same as for the ``Measurements`` field.
+
+  Default:
+
+  If he exclude field has not been defined no extra exclusions will be applied.
```

Comment:
Looks like a 't' is missing from the word 'the' here.

Reply:
Fixed